Satyam1923/Spring-Music-Player
# added preloader

### Description
It consists the code of adding a responsive preloader
- This PR does the following:
  - Adds preloader


### Related Issues
 Fixes: #143

### Changes
- Added a new feature to frontend 


### Testing Instructions
Detailed instructions on how to test the changes. Include any setup needed and specific test cases.
1. Pull this branch.
2. Run `npm install` to install dependencies.
3. Run `npm test` to execute the test suite.
4. Verify that preloader is working o not


### Checklist
Make sure to check off all the items before submitting. Mark with [x] if done.
